client/src/Router.rei

```diff
@@ -23,7 +23,8 @@ module Unload: {
    */
   module Provider: {
     let make:
-      React.childless => React.component(unit, React.noRetainedProps, unit);
+      ReactOld.childless =>
+      ReactOld.component(unit, ReactOld.noRetainedProps, unit);
```

Comment:
We should update these signatures to jsx3 as well, e.g.

```reason
[@react.component]
let make: unit => React.element;
```

Reply:
Changed.